### PR TITLE
Flatten enable-integration shortcode

### PR DESCRIPTION
**Pull request by AnouckColson**
## Why

The `enable-integration.html` shortcode has become too complex, hindering maintenance. 

## What's changed

- [x] Hardcode `Enable the integration` step on GitLab page.
- [x] Change `--repository` flag to `--server-project` for GitLab.
- [x] Hardcode `Enable the integration` step on GitHub page.
- [x] Hardcode `Enable the Cloud integration` step on Bitbucket page.

### CODE REVIEW

**Comment by AnouckColson** [comment]
#3562 needs to be merged so this one can pass.

File: sites/platform/src/integrations/source/bitbucket.md [hunk 1]
```diff
@@ -33,7 +33,71 @@ Copy the **Key** and **Secret** for your consumer.
...
+{{< codetabs >}}
+
++++
+title=Using the CLI
++++
+
+Run the following command:
+
+```bash
+platform integration:add \
```
Comment (chadwcarlson) [inline sites/platform/src/integrations/source/bitbucket.md:47]:
```suggestion
{{% vendor/cli %}} integration:add \
```

File: sites/platform/src/integrations/source/bitbucket.md [hunk 2]
```diff
@@ -33,7 +33,71 @@ Copy the **Key** and **Secret** for your consumer.
...
+- `PROJECT_ID` is the ID of your {{% vendor/name %}} project.
+- `OWNER/REPOSITORY` is the name of your repository in Bitbucket.
+- `CONSUMER_KEY` is is the key of the [OAuth consumer you created](#1-create-an-oauth-consumer).
+- `CONSUMER_SECRET` is the secret of the [OAuth consumer you created](#1-create-an-oauth-consumer).
+
+For example, if your repository is located at `https://bitbucket.org/platformsh/platformsh-docs`,
+the command is similar to the following:
+
+```bash
+platform integration:add \
```
Comment (chadwcarlson) [inline sites/platform/src/integrations/source/bitbucket.md:64]:
```suggestion
{{% vendor/cli %}} integration:add \
```

File: sites/platform/src/integrations/source/github.md [hunk 3]
```diff
@@ -54,7 +54,74 @@ generate and copy your token.
...
+{{< codetabs >}}
+
++++
+title=Using the CLI
++++
+
+Run the following command:
+
+```bash
+platform integration:add \
```
Comment (chadwcarlson) [inline sites/platform/src/integrations/source/github.md:68]:
```suggestion
{{% vendor/cli %}} integration:add \
```

File: sites/platform/src/integrations/source/github.md [hunk 4]
```diff
@@ -54,7 +54,74 @@ generate and copy your token.
...
+- `OWNER/REPOSITORY` is the name of your repository in GitHub.
+- `GITHUB_ACCESS_TOKEN` is the [token you generated](#1-generate-a-token).
+- `GITHUB_URL` is the base URL for your GitHub server if you self-host.
+   If you use the public `https://github.com`, omit the `--base-url` flag when running the command.
+
+For example, if your repository is located at `https://github.com/platformsh/platformsh-docs`,
+the command is similar to the following:
+
+```bash
+platform integration:add \
```
Comment (chadwcarlson) [inline sites/platform/src/integrations/source/github.md:86]:
```suggestion
{{% vendor/cli %}} integration:add \
```

File: sites/platform/src/integrations/source/gitlab.md [hunk 5]
```diff
@@ -35,7 +35,73 @@ and grants more permissions.
...
+{{< codetabs >}}
+
++++
+title=Using the CLI
++++
+
+Run the following command:
+
+```bash
+platform integration:add \
```
Comment (chadwcarlson) [inline sites/platform/src/integrations/source/gitlab.md:49]:
```suggestion
{{% vendor/cli %}} integration:add \
```

File: sites/platform/src/integrations/source/gitlab.md [hunk 6]
```diff
@@ -35,7 +35,73 @@ and grants more permissions.
...
+- `PROJECT/SUBPROJECT` is the name of your repository in GitLab.
+- `GITLAB_ACCESS_TOKEN` is the [token you generated](#1-generate-a-token).
+- `GITLAB_URL` is the base URL for your GitLab server if you self-host.
+   If you use the public `https://gitlab.com`, omit the `--base-url` flag when running the command.
+
+For example, if your repository is located at `https://gitlab.com/platformsh/platformsh-docs`,
+the command is similar to the following:
+
+```bash
+platform integration:add \
```
Comment (chadwcarlson) [inline sites/platform/src/integrations/source/gitlab.md:67]:
```suggestion
{{% vendor/cli %}} integration:add \
```

**Review by chadwcarlson — COMMENTED** [review]
Just CLI name vendorization - all good otherwise!

**Review by chadwcarlson — APPROVED** [review]

**Pull request merged**